docusaurus/docs/core-concepts/clients/tunnelers/02-windows.md

```diff
@@ -4,7 +4,44 @@
 
 The ZDEW has an installer which is available for download from GitHub.
 [Download the latest release here](https://github.com/openziti/desktop-edge-win/releases/latest).
-The installer will require administrator privileges to install successfully.
+
+This brings the user to the NetFoundry Ziti GitHub repository.  There are a list of releases, select the top release and the .exe package from the list of files.
+
+![image](/img/clients/windows-install1.jpg)
+
+**NOTE: Installer requires elevated permissions**. Right click on  Ziti.Desktop Edge Client installer file and select **Run as Administrator**.

```

Comment:
I think we can skip over how to run the installer and how to run the app. Those are the parts that seems self-explanatory to me.

Reply:
done